kamailio/db_scripts/kazoodb-sql.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh -e
+
```

Comment:
Why not to add comment about `DB_ENGINE` variable usage? Like this

#####################################################################################
##
## If you want prepare SQL file for PostgreSQL or MySQL server, then need to execute
## DB_ENGINE=postgres ./create-kazoodb-sql.sh
##
#####################################################################################